orchprovider/k8s/v1/k8s.go

```diff
@@ -1051,19 +1055,58 @@ func (k *k8sOrchestrator) createReplicaDeployment(volProProfile volProfile.Volum
 
 	glog.Infof("Adding replica(s) for Volume '%s'", vsm)
 
+	repLabelSpec := map[string]string {
+		string(v1.VSMSelectorKey):               vsm,
+		string(v1.PVCSelectorKey):               pvc,
+		string(v1.VolumeProvisionerSelectorKey): string(v1.JivaVolumeProvisionerSelectorValue),
+		string(v1.ReplicaSelectorKey):           string(v1.JivaReplicaSelectorValue),
+		// -- if manual replica addition
+		//string(v1.ReplicaCountSelectorKey):      strconv.Itoa(rCount),
+	}
+
+	//Add the application label to the replica deployment if it exists.
+	appLV := vol.Labels.ApplicationOld
+	if appLV != "" {
+		repLabelSpec[string(v1.ApplicationSelectorKey)] = appLV
+	}
+
+	//Set the Default Replica Topology Key ( kubernetes.io/hostname )
+	replicaTopoKey :=  v1.GetPVPReplicaTopologyKey(nil)
+
+	//One of the labels to match will always be a constant, which is 
```

Comment:
some the labels are meant to be deprecated. e.g. vsm & the ones which are not namespaced.
we should definitely make use of labels that start with `openebs.io/`.

Reply:
True. Probably better to raise a new PR that deprecates them. Since changing here, may require us to change in all other places, for labels like vsm.